.gitignore

```diff
@@ -1 +1 @@
-build/
+/gopath/
```

Comment:
There are 3 files in `/gopath/` that are tracked by git. I think putting tracked files in `.gitignore` is a bad practice.